[2025-02-08 12:38:34,582](run_tvsrv) Ошибка Инфотабло: возможно нет связи? HTTPConnectionPool(host='127.0.0.1', port=32109): Max retries exceeded with url: /ticket?p=8&t=%D0%A11 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff6b68f3a30>: Failed to establish a new connection: [Errno 111] Connection refused')). 
